### DISCUSSION TITLE
Documentation is unclear where I can get an IstioOperator?

### DISCUSSION THREAD

**Discussion by naxim92** [istio/istio#52053] [General]
Guys, I cannot follow where I can get an IstioOperator to use it terraform, not with istioctl

**Comment by zirain** [comment]
Are you saying IstioOperator controller/chart?

**Comment by naxim92** [comment]
Chart which helps me install IstiOperator controller and CRD

**Comment by zirain** [comment]
Istio didn't release it for a long time, [it's not recommond by the community](https://istio.io/latest/docs/setup/install/operator/), and will be [deprecated](https://github.com/istio/enhancements/issues/166) in the future.
You can package it yourself, if you really want it.
